### ISSUE TITLE
Personen - Begriffe im Menü "Authentication Code"

### ISSUE THREAD

**Issue by alexandermarxen** [rism-international/tasks#636]
Englische Oberfläche:
Im Auswahl "Authentication Code" gibt es derzeit die Begriffe "individualized" und "not individualized". Bitte wie folgt ändern:
individualized -> differentiated
not individualized -> undifferentiated

**Comment by HirschSt** [comment]
Wenn die Begriffe geändert werden, müssen zusätzlich möglichst synchron auch alle Daten bei den Personen abgeändert werden, d.h. die Änderung muss sowohl in Muscat als auch über Wartungsskripte vollzogen werden.

**Comment by alexandermarxen** [comment]
Oh ja, das stimmt natürlich. Soll ich ein Extra-Ticket dafür schreiben?

**Comment by HirschSt** [comment]
Nein, wir müssen das nur im Kopf behalten, wenn wir die Umstellung machen.
Ich werde das vermutlich nächste Woche vorbereiten.

Am Mo., 10. Feb. 2020 um 09:54 Uhr schrieb alexandermarxen <
notifications@github.com>:

**Comment by jenniferward** [comment]
Erledigt mit https://github.com/rism-digital/muscat/issues/1499
